src/ray/gcs/client_test.cc

```diff
@@ -255,6 +256,170 @@ TEST_F(TestGcsWithAsio, TestLogAppendAt) {
   TestLogAppendAt(job_id_, client_);
 }
 
+void TestDeleteKeysFromLog(const JobID &job_id,
+                           std::shared_ptr<gcs::AsyncGcsClient> client,
+                           std::vector<std::shared_ptr<ObjectTableDataT>> &data_vector,
+                           bool use_one_id) {
```

Comment:
Can we get rid of `use_one_id`, and pass in a data_vector with only one element?

Reply:
Log can be append with many elements. If we remove that, the deletion test only covers the case with one elements. However, that may be fine.